aerleon/lib/cisco.py

```diff
@@ -475,7 +428,6 @@ def __init__(
         platform='cisco',
         verbose=True,
     ):
-        super().__init__(term)
```

Comment:
Why does `Cisco.Term.__init__` no longer call `super().__init__`? This causes UnsupportedFilterError checks to be skipped and stops ProtocolNameToNumber() from being called on term.protocol.

https://github.com/aerleon/aerleon/blob/2820a3920238460d2143224ee6d4cc8c1eae2b27/aerleon/lib/aclgenerator.py#L128-L139

Reply:
Added this back, I believe it was giving me issues at one point.